src/model/table.py

```diff
@@ -48,51 +70,40 @@ def __get_efficient_index__(self, query):
 
     def __get_all_primary_keys__(self):
         primary_keys = []
-        for primary_key_path in os.listdir(self.__get_data_path__()):
-            primary_key = self.__get_primary_key_from_path__(primary_key_path)
-            primary_keys.append(primary_key)
+        for primary_key in os.listdir(self.get_data_path()):
+            if primary_key != "Lock":
```

Comment:
Again, u should have directory with name table name

Sub directories : data, lock, indices